extensions/2.0/Vendor/EXT_textureInfo_constant_lod/schema/textureInfo.EXT_textureInfo_constant_lod.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "EXT_textureInfo_constant_lod TextureInfo Extension",
+  "type": "object",
+  "description": "Enables dynamically calculated UV coordinates to keep the texture near a constant level of detail",
+  "allOf": [
+    { "$ref": "glTFProperty.schema.json" }
+  ],
+  "properties": {
+    "repetitions": {
+      "type": "number",
+      "description": "The number of times the texture is repeated per meter.",
+      "minimum": 1.0,
+      "default": 1.0
+    },
+    "offset": {
+      "type": "array",
+      "description": "The offset in meters used to shift the texture.",
+      "items": {
+        "type": "number"
+      },
+      "minItems": 2,
+      "maxItems": 2,
+      "default": [ 0.0, 0.0 ]
+    },
+    "minClampDistance": {
+      "type": "number",
+      "description": "The minimum distance in meters from the eye to the surface at which to clamp the texture.",
+      "minimum": 0.0,
+      "default": 1.0
+    },
+    "maxClampDistance": {
+      "type": "number",
+      "description": "The maximum distance in meters from the eye to the surface at which to clamp the texture.",
+      "minimum": 0.0,
+      "default": 4294967296.0
+    },
+    "extensions": { },
+    "extras": { }
```

Comment:
Unless all properties are optional, the schema should specify which ones are required.